testlib/src/test/java/io/spine/testing/VerifyShould.java

```diff
@@ -1236,18 +1181,16 @@ public void run() {
     public void pass_assertThrowsWithCause_if_runnable_throws_specified_exception_with_specified_cause() {
         Throwable cause = new EmptyStackException();
         RuntimeException runtimeException = new RuntimeException(cause);
-        Runnable throwsRuntimeException = new Runnable() {
-            @Override
-            public void run() {
-                throw runtimeException;
-            }
+        Runnable throwsRuntimeException = () -> {
+            throw runtimeException;
         };
 
         assertThrowsWithCause(runtimeException.getClass(), cause.getClass(),
                                      throwsRuntimeException);
     }
 
-    @SuppressWarnings("EqualsAndHashcode")
+    @SuppressWarnings({"EqualsAndHashcode", "EqualsHashCode"})
```

Comment:
I have looked at the issue, and it seems that there is no good way to eliminate this duplication. For some reason JetBrains and ErrorProne have different notion, and it seems non-customizable: in ErrorProne the warning name is [a part of their source code](https://github.com/google/error-prone/blob/master/core/src/main/java/com/google/errorprone/bugpatterns/EqualsHashCode.java#L60), and in IDEA I have found no option to override warning names as well.

So the only way is to have `hashCode()` :)